catalogs/catalog-kafka/src/main/java/com/datastrato/gravitino/catalog/kafka/KafkaCatalogOperations.java

```diff
@@ -132,8 +134,16 @@ public void initialize(Map<String, String> config, CatalogInfo info) throws Runt
         AdminClientConfig.CLIENT_ID_CONFIG,
         String.format(CLIENT_ID_TEMPLATE, config.get(ID_KEY), info.namespace(), info.name()));
 
+    try {
+      adminClient = AdminClient.create(adminClientConfig);
```

Comment:
What's the timeout?  The web page has hang here for a long time if I use a wrong broker address.
![image](https://github.com/datastrato/gravitino/assets/15794564/fb19bcb5-0871-4f8d-ac21-dc39d7cd71f9)

Reply:
Is it a UI problem or a backend problem?

Reply:
Backend. From the backend log, the list opearation block(retry) for a long time.

Reply:
It's because you are listing all the topics, the default API timeout is 1 minute, should I set it to a shorter time?

Reply:
I prefer to shorten the time here.

Reply:
Should we create a new issue to track this? Since is already a list topics operation, not a create catalog issue

Reply:
> Should we create a new issue to track this? Since is already a list topics operation, not a create catalog issue

Agree, a separate issue may be better.